src/encoded/tests/test_schema_experiment.py

```diff
@@ -107,3 +109,14 @@ def test_experiment_valid_statuses(status, testapp, experiment):
 def test_experiment_invalid_statuses(status, testapp, experiment):
     with pytest.raises(Exception):
         testapp.patch_json(experiment['@id'], {'status': status})
+
+
+def test_submission_date_dependency(testapp, experiment_no_error):
+    expt = testapp.post_json('/experiment', experiment_no_error).json['@graph'][0]
+    res = testapp.patch_json(expt['@id'], {
+        'status': 'submitted'}, expect_errors=True)
+    assert res.status_code == 422
+    res = testapp.patch_json(expt['@id'], {
+        'status': 'submitted',
+        'date_submitted': '2000-10-10'}, expect_errors=True)
```

Comment:
Why do we expect_errors here?

Reply:
Also I think you can use this idiom instead of the assert res.status_code: 
`testapp.patch_json(expt['@id'], {'status': 'submitted'}, status=422)`
`testapp.patch_json(expt['@id'], { 'status': 'submitted', 'date_submitted': '2000-10-10'}, status=200)`

Reply:
Change name to test_experiment_blah_blah. Also I think you reverted the changes that cleaned this up. i.e. `testapp.patch_json(expt['@id'], {'status': 'submitted'}, status=422)`.